app/main.dev.ts

```diff
@@ -12,23 +12,9 @@ import path from 'path';
 import { app, BrowserWindow, ipcMain } from 'electron';
 import MenuBuilder from './menu';
 import { LedgerServer } from './ledgerIpc';
-import { init, showReportDialog } from '@sentry/electron';
 
 app.name = 'Ambrosia';
 
-init({
-  dsn:
-    'https://c4bbd637d7154e338433faf792b7c320@o461938.ingest.sentry.io/5471443',
-  beforeSend(event) {
```

Comment:
Did you remove this on purpose from the other initialization?

Reply:
Yeah, that was on purpose. I removed `sentry-electron` and am only using the `sentry-react`.